pages/create-video/components/AlignEnd.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const container: React.CSSProperties = {
+  alignSelf: "flex-end",
+};
+
+export const AlignEnd: React.FC<{
+  children: React.ReactNode;
+}> = ({ children }) => {
+  return <div style={container}>{children}</div>;
+};
```

Comment:
I don't see this component used anywhere is it necessary?

Reply:
Will remove, was part of boilerplate.